package.json

```diff
@@ -21,9 +21,10 @@
     "test:features": "cross-env CACHE=true FEATURES=true karma start config/karma/karma.config.features.js",
     "test:playground": "karma start config/karma/karma.config.transformer.playground.js",
     "test:playground:build": "karma start config/karma/karma.config.transformer.playground.build.js",
-    "version": "conventional-changelog -p angular -i CHANGELOG.md -s -r -0",
-    "copyForPublish": "cp -rf package.json README.md CHANGELOG.md dist",
-    "preparePublish": "npm run build && npm run copyForPublish"
+    "release": "semantic-release",
```

Comment:
I read the documentation briefly. This command -> semantic-release -> will automatically bump the version, create the tags and canghe the changelog file right and then it will create a commit `ci(release): new version ${nextRelease.version}`?

When should be using commitmsg and cz?

Reply:
The `release` script behavior is based on the `.releaserc.json` configuration file.
So yes indeed, it does all the things you said plus it create an npm release and a GitHub release.

As indicated in the "What to do next", there is one possible feature to add to check that the commit messages are well written. This is important since commitizen is configured to define how do you wish to write the commits and semantic uses it to define then based on the commits if the next release should be a patch, a minor or a major version. The `commitmsg` in this case is used but as now it can be removed since I did not configure the hook.

Finally, the `cz` script is a wonderful CI tool once again based on the commitizen configuration provided to write the commits without possible errors. This is a mix between manual commit messages and linted commit messages with a hook. People used to the commitizen convention are well aware of the cz script to use as an alternative and is very welcoming to new contributors since they just do not need to read the contributing guide to know how to right a good commit since the CLI got them covered.

Reply:
Removed all the configuration related to the git hook for now.